common/v2/features/SwapAssets/stateFactory.tsx

```diff
@@ -43,7 +43,10 @@ const swapFlowInitialState = {
   txConfig: undefined,
   rawTransaction: undefined,
   dexTrade: undefined,
-  txReceipt: undefined
+  txReceipt: undefined,
+  initialValue: undefined,
+  initialRate: undefined,
```

Comment:
- why 'initial' ? Do we expect others in the future? 
- does 'value' correspond to 'fromAmount' or 'toAmount'? Please stay consistant with the naming.